app/src/main/java/com/example/habitsmasher/HabitList.java

```diff
@@ -14,12 +14,32 @@
     }
 
     public void addHabit(String title, String reason, Date date) {
-        Habit newHabit = new Habit(title, reason, date);
+        Habit newHabit = new Habit(title, reason, date, 0);
         _habits.add(newHabit);
         //addHabitToDatabase(title, reason, date);
     }
     public void addHabit(Habit habit) {
         _habits.add(habit);
         //addHabitToDatabase(title, reason, date);
     }
+
+    /**
+     * Method that edits the habit at position pos
+     * @param title New title of habit
+     * @param reason New reason of habit
+     * @param date New date of habit
+     * @param pos Position of habit in the HabitList
+     */
+    public void editHabit(String title, String reason, Date date, int pos) {
+        try {
+            Habit habit = _habits.get(pos);
```

Comment:
Instead of `_habits.get(pos)` should we be doing `_habits.getHabitList().get(pos)` instead?